src/test/java/jewellery/inventory/integration/SaleCrudIntegrationTest.java

```diff
@@ -473,4 +662,88 @@ private ResponseEntity<ResourceReturnResponseDto> getResourceReturnResponse(
         null,
         ResourceReturnResponseDto.class);
   }
+
+  private Organization createOrganizationInDatabase(
+      OrganizationRequestDto testOrganizationRequest) {
+    ResponseEntity<Organization> response =
+        this.testRestTemplate.postForEntity(
+            getBaseOrganizationsUrl(), testOrganizationRequest, Organization.class);
+
+    assertEquals(HttpStatus.CREATED, response.getStatusCode());
+    return response.getBody();
+  }
```

Comment:
We have a similar method in OrganizationCrudIntegrationTest -> `createOrganizationsWithRequest`

Reply:
Here I think not to change anything. Later I think to make some integration test helpers to reuse methods from.